sources/packages/backend/apps/db-migrations/src/sql/Student/Add-col-disability-status.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS disability_status sims.disability_status;
+
+COMMENT ON COLUMN sims.students.disability_status IS 'Disability status of the student.';
+
+-- Update the disability_status column based on pd_verified and pd_date_sent for existing data.
+UPDATE
+    sims.students
+SET
+    disability_status = CASE
+        WHEN pd_verified IS NULL
+        AND pd_date_sent IS NOT NULL THEN 'Requested'
+        WHEN pd_verified = true THEN 'PD'
+        WHEN pd_verified = false THEN 'Declined'
+        ELSE 'Not requested' :: sims.disability_status
+    END;
+
+-- Set not null constraint after updating the column.
+ALTER TABLE
+    sims.students
+ALTER COLUMN
+    disability_status
+SET
```

Comment:
since, its not null, how about adding the default as `NotRequested`?

Reply:
I had the same thought initially. But because due to the reason that we manually set a default value from the code, I preferred not to have a default at SQL as well. 

![image](https://github.com/bcgov/SIMS/assets/54600590/32419ac4-b8ff-4ecd-9e1f-4f1dc13db9ca)

![image](https://github.com/bcgov/SIMS/assets/54600590/543dda87-f901-42ae-993b-9752cf80f4ff)

Reply:
I feel, its good this way, rather setting it as a default value in the SQL.